test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1956,42 +1956,70 @@ def forward(self, input):
         x = torch.ones(5, 6)
         self.run_test(DimArange(), x)
 
+    def _test_compare_ops(self, model, num_inputs):
```

Comment:
Can you add a couple more tests for combination of input types (like float and int)?